packages/round-manager/src/features/round/RoundDetailForm.tsx

```diff
@@ -895,6 +909,206 @@ function MatchingCap(props: {
+                      {props.supportTypes.map(
+                        (type) =>
+                          !type.default && (
+                            <Listbox.Option
+                              key={type.name}
+                              className={({ active }) =>
+                                classNames(
+                                  active
+                                    ? "text-white bg-indigo-600"
+                                    : "text-gray-900",
+                                  "relative cursor-default select-none py-2 pl-3 pr-9"
+                                )
+                              }
+                              value={type.name}
+                              data-testid="support-type-option"
+                            >
+                              {({ selected, active }) => (
+                                <>
+                                  <div className="flex items-center">
+                                    <span
+                                      className={classNames(
+                                        selected
+                                          ? "font-semibold"
+                                          : "font-normal",
+                                        "ml-3 block truncate"
+                                      )}
+                                    >
+                                      {type.name}
+                                    </span>
+                                  </div>
+
+                                  {selected ? (
+                                    <span
+                                      className={classNames(
+                                        active
+                                          ? "text-white"
+                                          : "text-indigo-600",
+                                        "absolute inset-y-0 right-0 flex items-center pr-4"
+                                      )}
+                                    >
+                                      <CheckIcon
+                                        className="h-5 w-5"
+                                        aria-hidden="true"
+                                      />
+                                    </span>
+                                  ) : null}
+                                </>
+                              )}
+                            </Listbox.Option>
+                          )
+                      )}
+                    </Listbox.Options>
+                  </Transition>
+                </div>
+                {props.errors.roundMetadata?.support?.type && (
+                  <p className="mt-2 text-xs text-pink-500">
+                    {
+                      "You must select a support type."
+                      // TODO: Use YUP for error message
```

Comment:
Should we update this right now to keep it consistent ?

Reply:
Yes - however I could not for the life of me figure out why it was not working as intended. Perhaps the nested yup objects require a different approach?

Reply:
@chibie got any ideas here ?  you've got a solid understanding of how yup works